files/templates/mbta/login/messages/messages_en.properties

```diff
@@ -23,6 +52,8 @@ message.info=Please be aware of the following:
 
 registration.forbidden.username=Please <a href="{0}">click here</a> to log in as MBTA Staff
 registration.forbidden.email=Please <a href="{0}">click here</a> to log in as MBTA Staff
+registration.bad.format.mobile=Your phone number must have ten digits.
```

Comment:
question: is the `mobile` part of this the name of the attribute? does it need to be updated to `phone_number`?